test/factory/ProxyFactory.spec.ts

```diff
@@ -91,17 +94,29 @@ describe("ProxyFactory", async () => {
             expect(await singleton.masterCopy()).to.be.eq(AddressZero)
             expect(await hre.ethers.provider.getCode(proxyAddress)).to.be.eq(await factory.proxyRuntimeCode())
         })
+
+        it('should not be able to deploy same proxy twice', async () => {
+            const { factory, singleton } = await setupTests()
+            const initCode = singleton.interface.encodeFunctionData("init", [])
+            const proxyAddress = await calculateProxyAddress(factory, singleton.address, initCode, saltNonce)
+            await expect(
+                factory.createProxyWithNonce(singleton.address, initCode, saltNonce)
+            ).to.emit(factory, "ProxyCreation").withArgs(proxyAddress, singleton.address)
+            await expect(
+                factory.createProxyWithNonce(singleton.address, initCode, saltNonce)
+            ).to.be.revertedWith("Create2 call failed")
+        })
     })
 
-    describe("createProxyWithNonce", async () => {
+    describe("createChainSpecificProxyWithNonce", async () => {
 
-        const saltNonce = 42
+        const saltNonce = 73
```

Comment:
why this change?

Reply:
It's not a change technically but new code. This is the value I intended to use within `createChainSpecificProxyWithNonce` tests. I just like the number 😂 I can change it back

Reply:
I like 42 (was chosen on purpose :D)

Reply:
What's the purpose out of interest?

Reply:
I changed it back to 42 @rmeissner

Reply:
it is the answer to life the universe and everything